docs/usecases/extending_data_ecosystem.rst

```diff
@@ -0,0 +1,151 @@
+.. _usecase_extending_data_ecosystem:
+
+Extending the data ecosystem
+----------------------------
+
+This use case sketches the basic steps to gain access to existing datasets
+distributed across the `Globus <https://www.globus.org/>`_ network via Datalad
+and git-annex. It elaborates on
+
+#. How to enable the git-annex `Globus special remote  <https://github.com/CONP-PCNO/git-annex-remote-globus>`_
+   to access files content from `Globus.org <https://www.globus.org/>`_ ,
+#. How disk-space aware computing can be eased by DataLad and git-annex
+#. How to work with large datasets distributed across systems avoiding unnecessary
+   replication and maintaining a light-weight approach
+
+
+The Challenge
+^^^^^^^^^^^^^
+Every day, researchers from different fields strive to advance present
+state-of-the-art scientific knowledge by generating and publishing novel
+results. Crucially, they must share such results with the scientific
+community to enable other researchers to further build on existing data
+and avoid duplicating work.
+
+Therefore, it is common practice for researchers to both download and
+publish research content in a dataset format via a publicly available
+portal, the `Canadian Open Neuroscience Portal (CONP) <https://conp.ca/>`_,
+which provides them with a vast dataset repository. Managing the scientific data
+ecosystem is extremely challenging given the amount of new data generated
+every day. CONP must take a strategic solution to allow researchers to dynamically
+work on present data, upload new versions and generate additional work. Such
+system must be flexible, dynamic and light-weight. It would be very useful to distribute
+datasets across multiple locations are reduce the need of replicating data
+
+
+The Datalad Approach
```

Comment:
here and elsewhere, DataLad (not Datalad).  I will submit a PR to tune up some missed hits in other docs